scripts/01-import_and_maxfilter.py

```diff
@@ -74,6 +75,37 @@ def init_dataset():
     _write_json(fname, ds_json, overwrite=True)
 
 
+def get_mf_cal_fname(subject, session):
+    if config.mf_cal_fname is None:
+        mf_cal_fpath = BIDSPath(subject=subject,
+                                session=session,
+                                acquisition='calibration',
+                                suffix='meg',
+                                extension='dat',
+                                root=config.bids_root).fpath
```

Comment:
You should use [`BIDSPath. meg_calibration_fpath `](https://mne.tools/mne-bids/stable/generated/mne_bids.BIDSPath.html#mne_bids.BIDSPath.meg_calibration_fpath)